client/src/components/ShoeShowPage.js

```diff
@@ -14,30 +16,32 @@ const ShoeShowPage = props => {
               }
             const shoeData = await response.json()
             setShoe(shoeData.shoe)
+            setReviews(shoeData.shoe.reviews)
```

Comment:
this second piece of state should ideally be removed, because line 18 above it is already setting all of the `shoe` properties in state, including reviews